build.sbt

```diff
@@ -101,6 +101,25 @@ lazy val chiselFrontend = (project in file("chiselFrontend")).
   dependsOn(coreMacros)
 
 lazy val chisel = (project in file(".")).
+  enablePlugins(BuildInfoPlugin).
+  settings(
+    // We should really be using name.value, but currently, the package is "Chisel" (uppercase first letter)
+    buildInfoPackage := /* name.value */ "chisel3",
+    buildInfoOptions += BuildInfoOption.BuildTime,
+    buildInfoKeys := Seq[BuildInfoKey](buildInfoPackage, version, scalaVersion, sbtVersion),
+    // Move the managed source directory where git won't complain about it,
```

Comment:
The plugin documentation https://github.com/sbt/sbt-buildinfo says the generated source is under `target/`, which should be ignored already? And as long as it doesn't drop the file in one of the existing source folders (is it?), adding an entry in .gitignore may be a better solution?

Also, this comment block needs reformatting. And what's up with the commented out line of code?